nemo/core/config/schedulers.py

```diff
@@ -51,6 +51,20 @@ class WarmupHoldSchedulerParams(WarmupSchedulerParams):
     min_lr: float = 0.0
 
 
+@dataclass
+class WarmupAnnealingHoldSchedulerParams(SchedulerParams):
```

Comment:
Why not just extend `WarmupSchedulerParams(SchedulerParams)` and drop max_steps, warmup_steps and warmup_ratio